### ISSUE TITLE
Issue with DS and Malisis Doors

### ISSUE THREAD

**Issue by vpontin** [OreCruncher/DynamicSurroundings#258]
*Mod Version:*
- DS - 3.4.9.4
- MalisisDoors-1.12.2-7.3.0 (MalisisCore 6.4.0)

*Forge Version:*
- b2618

*Description:*
- Basically a duplicate of https://github.com/Ordinastie/MalisisDoors/issues/237 : Basically when i right-click to open a door the left-click sound is played. According to @OreCruncher its a DS problem, so i've created the proper ticket here.

**Comment by OreCruncher** [comment]
I can't figure a solution for this one.  May be a bit.

**Comment by vpontin** [comment]
So this is more related to door animation (regardless of Vanilla or Malisis)?

**Comment by OreCruncher** [comment]
Not the animation itself, but the hitbox of the door when it is open/closed.  If you stand in front of a closed door your look vector (your line of sight) intersects with the door hit box.  When you open the door, the hit box changes so all that is facing you is the door edge.  That edge is now the hitbox.  Your line of sight now pass through that block because it doesn't "hit" that box.

Note that this applies to any block.  You can get the same effect breaking a log with an axe.  If there is nothing behind the log when it breaks you get the axe swing sound.

**Comment by darthvader45** [comment]
Yeah, sometimes get that when mining.  Hopefully there's a way to fix this, preferably without being TOO hacky.
